Lab 2.ipynb

```diff
@@ -1434,36 +446,86 @@
     "# X is chosen background Gaussian width\n",
     "# Y is chosen signal strength\n",
     "X = 1.7\n",
-    "Y = 5.6\n",
+    "Y = 8.2\n",
+    "num_pixels = 10000\n",
     "\n",
     "k = np.linspace(-10, 10, 1000)\n",
     "background = stats.norm.pdf(k, loc = 0, scale = X)\n",
-    "comb = background\n",
+    "background = background / np.sum(background)\n",
+    "full_run = background * num_pixels\n",
     "\n",
-    "for i in range(1000):\n",
-    "    print(i)\n",
-    "    comb = np.convolve(comb, background)\n",
-    "    comb = comb / np.sum(comb)\n",
-    "k_c = np.linspace(-10, 10, len(comb))\n",
     "\n",
     "fig, ax = plt.subplots(1, 1)\n",
     "ax.set_title(\"Background Gaussian\")\n",
     "ax.set_xlabel(\"Number of photons\")\n",
     "ax.set_ylabel(\"Probability\")\n",
```

Comment:
It's not the probability anymore because we scaled it, I think it's the number of instances now